platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -306,6 +313,80 @@ private void configureConduit(ClientConfiguration clientConfig) {
                 }
             }
         }
+
+        TLSClientParameters tlsParams = httpConduit.getTlsClientParameters();
+        if (tlsParams == null) {
+            tlsParams = new TLSClientParameters();
+        }
+
+        tlsParams.setDisableCNCheck(disableCnCheck);
+
+        tlsParams.setUseHttpsURLConnectionDefaultHostnameVerifier(true);
+        tlsParams.setUseHttpsURLConnectionDefaultSslSocketFactory(true);
+        tlsParams.setCipherSuites(Arrays.asList(System.getProperty("https.cipherSuites")
+                .split(",")));
+
+        KeyStore keyStore = null;
```

Comment:
I hate this block of code - I wish it could be extracted to methods for `getKeyManagers()` and `getTrustManagers()` so the sets on `tlsParams` would be more clearly delineated - but then you'd end up with two nearly identical private methods and that's no better.